### PR TITLE
Add Maintainter guide

### PR DESCRIPTION
**Pull request by sapegin**
Feel free to comment and ask more questions if anything isn't clear.

### CODE REVIEW

File: docs/Maintenance.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+<!-- tocstop -->
+
+_See also [Developer guide](Development.md)._
+
+## We need you!
+
+Help us develop and maintain Styleguidist:
+
+* Answer questions in [GitHub issues](https://github.com/styleguidist/react-styleguidist/issues) and [Stack Overflow](https://stackoverflow.com/questions/tagged/react-styleguidist).
```
Comment (bluetidepro) [inline docs/Maintenance.md]: Should we also add the Gitter chat as a place to answer questions in?

File: docs/Maintenance.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* Review [pull requests](https://github.com/styleguidist/react-styleguidist/pulls).
+* Fix bugs and add new features.
+* Write articles and talk about Styleguidist on conferences and meetups (we’re always happy to review your texts and slides).
+
+## Commit message conventions
+
+[Commit message conventions](https://github.com/tamiadev/semantic-release-tamia/blob/master/Convention.md) are based on Angular conventions.
```
Comment (bluetidepro) [inline docs/Maintenance.md]: Is there any tools that allow us to like lint this or validate it on an automation level? May make it easier to enforce? Or maybe a "precommit" hook?

File: docs/Maintenance.md (hunk 1 above)
Comment (sapegin) [reply docs/Maintenance.md]: I’m actually thinking about removing all the Gitter links — changes to have an answer there are very small, and questions are usually of very low quality, so you they can't have an answer without asking 10 more questions ;-)

File: docs/Maintenance.md (hunk 2 above)
Comment (sapegin) [reply docs/Maintenance.md]: We don't care what contributors write in commit messages, or how many they add to a pull request ;-) So the only way to make it actually works is to do squash-merge and rewrite messages.

**Pull request merged**